ssn/index.html

```diff
@@ -612,6 +612,28 @@ <h3>IP68 Smart Sensor</h3>
         <pre class="example" data-include="./rdf/examples/2017ex25.ttl" data-include-format="text"></pre>
       </div>
     </section>
+
+    <section id="InkBird-IBS-TH2">
+      <h3>InkBird IBS TH2 Sensor</h3>
+      <div>
+        <p>These examples describe a commercially available platform, the IBS TH2, marketed under the InkBird brand.</p>
+        
+        <h4>Recording survival and operating ranges for a sensor</h4>
+        
+        <p>In this example we represent the capability of the platform's temperature sensor to measure ambient air and the physical
```

Comment:
"Temperature sensor of the platform". I heard that "'s" only applies to human beings, but I am not a native.